releasewarrior/helpers.py

```diff
@@ -8,6 +8,9 @@
 
 logger = logging.getLogger('releasewarrior')
 
+BUG_RE = r'(?<!\[)\b[bB][uU][gG]\s*(\d+)(?!\]\(http)\b'  # Don't match already linked bugs
```

Comment:
If you had [bug NNN] it would work still! the \(http is to prevent manually created links from [bug NNN](link) from getting doubly wrapper (negative look ahead)